pysatModels/tests/test_utils_extract.py

```diff
@@ -14,22 +13,21 @@
 import pysat
 from pysat.instruments import pysat_testmodel
 
-import pysatModels as ps_mod
 import pysatModels.utils.extract as extract
 
 
-@pytest.mark.skip("input requires a regular grid for the model")
-class TestUtilsExtractModObs(object):
-    """Unit tests for `utils.extract.extract_modelled_observations`."""
+class TestUtilsExtractInstThroughMod(object):
+    """Unit tests for `instrument_view_through_model`."""
 
     def setup(self):
         """Set up the unit test environment for each method."""
 
         self.inst = pysat.Instrument(platform='pysat', name='testing')
-        self.model = pysat.Instrument(inst_module=pysat_testmodel)
-        self.inst.load(date=pysat_testmodel._test_dates[''][''])
-        self.model.load(date=pysat_testmodel._test_dates[''][''])
-        self.model_label = 'tmodel'
+        self.model = pysat.Instrument(inst_module=pysat_testmodel, tag='')
+        self.inst.load(date=pysat_testmodel._test_dates[''][''],
+                       use_header=True)
+        self.model.load(date=pysat_testmodel._test_dates[''][''],
+                        use_header=True)
         self.input_args = [self.inst, self.model.data,
                            ["longitude", "latitude", "altitude"],
                            ["longitude", "latitude", "altitude"],
```

Comment:
```suggestion
                           ["latitude", "longitude", "altitude"],
```

Reply:
The order of the dimensions needs to match the ordering within xarray.

Reply:
Does it raise an error when it happens?